share/spice/npm/npm.js

```diff
@@ -2,96 +2,55 @@
+        
 
         Spice.add({
             id: "npm",
             name: "Software",
-            data: api_result,
+            data: api_result.results,
             meta: {
                 sourceName: "npmjs",
-                sourceUrl: 'http://npmjs.org/package/' + api_result.name
+                itemType: (api_result.total === 1) ? 'npm package' : 'npm packages',
+                sourceUrl: 'https://www.npmjs.com/search?q=' + query
             },
             normalize: function(item) {
-                var boxData = [{heading: 'Package Information:'}];
-
-                if (item.author) {
-                    boxData.push({
-                        label: "Author",
-                        value: item.author.name
-                    });
-                }
-
-                if (item.homepage) {
-                    boxData.push({
-                        label: "Project Homepage",
-                        value: item.homepage,
-                        url: item.homepage
-                    });
-                }
-
-                if (item.license) {
-                    boxData.push({
-                        label: "License",
-                        value: item.license
-                    });
-                }
-
-                if (item.repository) {
-                    boxData.push({
-                        label: "Repository",
-                        value: item.repository.url
-                    });
-                }
-
-                if (item.engines) {
-                    boxData.push({
-                        label: "Engines",
-                        value: item.engines.node
-                    });
-                }
-
-                if (item.dist) {
-                    boxData.push({
-                        label: "Source",
-                        value: item.dist.tarball,
-                        url: item.dist.tarball
-                    });
-                }
-
-                if (item.dependencies) {
-                    var dependencies = $.map(item.dependencies, function(val, key) {
-                        return key + " (" + val + ")";
-                    });
-
-                    if (dependencies.length > 0) {
-                        boxData.push({
-                            label: "Dependencies",
-                            value: dependencies.join(", ")
-                        });
-                    }
-                }
-
+                
+                var publisher = item.package.publisher.username;
+                
                 return {
-                    title: item.name + " " + item.version,
-                    subtitle: item.description,
-                    infoboxData: boxData,
+                    title: item.package.name,
+                    subtitle: "version: " + item.package.version,
+                    description: item.package.description,
+                    url: item.package.links.npm,
+                    rating: item.score.final.toFixed(2),
```

Comment:
@adampolar it seems like we should be sorting results in order of decreasing rating?

I'm wondering if this is actually something we need to show to the user though because its an npms.io concept and it may not be clear what "score" means or where it's coming from.

For example, https://npms.io/search?q=wikipedia seems to have a 0.06 score before a 0.66 -- that seems like a sorting error we can handle on our end?